application/btsieve/src/route_factory.rs

```diff
@@ -25,24 +24,21 @@ pub trait QueryType {
     fn route() -> &'static str;
 }
 
-pub trait ExpandResult {
+pub trait ToHttpPayload<Q, R> {
```

Comment:
:question: Where is the `Q` type parameter used?

Reply:
It is not used at all but without it, rustc cannot infer how this method is used in the code because the implementation is not bound to the query :(

Reply:
Thanks for explaining! I think the type parameter is fine :grinning: